alerts.tf

```diff
@@ -147,15 +130,14 @@ resource "newrelic_nrql_alert_condition" "high_latency_urgent" {
   }
 }
 
-# non-urgent conditions
-
-resource "newrelic_nrql_alert_condition" "error_rate_4xx" {
-  count = var.alert_error_rate_4xx_enable ? 1 : 0
+# urgent/non urgent conditions
```

Comment:
I think we can remove these section titles, since the distinction is not so clear anymore.